vm/Vagrantfile

```diff
@@ -25,12 +25,6 @@ Vagrant.configure(2) do |config|
     #--stroagectl can be "IDE Controller" in other Operating Systems
   end
 
- # global configuration
- config.vm.network :forwarded_port, guest: 22, host: 2222, id: "ssh", disabled: true
- config.vm.network :forwarded_port, guest: 22, host: 2223, id: "ssh", auto_correct: true
- # for 'vagrant ssh'
- config.ssh.port = 2223
-
```

Comment:
With these lines present in the Vagrantfile, `vagrant up` on my macOS 10.14.6 system using vagrant version 2.2.5 'hung' shortly after this step, probably trying to ssh into the VM using the default port 22, but failing.  At least, that is my guess why it was failing, since commenting out these lines caused it to proceed past that failing step.